### DISCUSSION TITLE
2006/05/12/%E6%89%93%E5%BC%80-hotmail-%E4%BF%A1%E4%BB%B6%E7%9A%84%E6%96%B9%E6%B3%95

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#83] [Announcements]
# 2006/05/12/%E6%89%93%E5%BC%80-hotmail-%E4%BF%A1%E4%BB%B6%E7%9A%84%E6%96%B9%E6%B3%95

最近 有些地方无法打开 hotmail，我们公司就不行。

https://blog.qizhen.xyz/2006/05/12/%E6%89%93%E5%BC%80-hotmail-%E4%BF%A1%E4%BB%B6%E7%9A%84%E6%96%B9%E6%B3%95

**Comment by ruanqizhen** [comment]
之前博客下的留言：


奕瑋
2006年05月13日 14:11 编辑
我就知道阮主一定有办法的,先行谢过!
